src/view/uniprotkb/components/GeneNamesView.tsx

```diff
@@ -2,13 +2,23 @@ import React, { Fragment } from 'react';
 import { v1 } from 'uuid';
 import { InfoList } from 'franklin-sites';
 import { GeneNamesData } from '../../../model/uniprotkb/sections/NamesAndTaxonomyConverter';
+import { NameWithEvidence } from './ProteinNamesView';
+import { ValueWithEvidence } from '../../../model/types/modelTypes';
 
 export const geneAlternativeNamesView = (
-  alternativeNames: { value: string }[],
+  alternativeNames: ValueWithEvidence[],
   firstComma: boolean = true
 ) => {
-  const altNames = alternativeNames.map(altName => altName.value).join(', ');
-  return `${firstComma ? ', ' : ''}${altNames}`;
+  return (
+    <Fragment>
+      {firstComma && ', '}
+      {alternativeNames
+        .map<React.ReactNode>(altName => (
+          <NameWithEvidence data={altName} key={altName.value} />
+        ))
+        .reduce((prev, curr) => [prev, ', ', curr])}
```

Comment:
I am now seeing a an extra space after the first alternative name: `CARD15 , IBD1`

![Screenshot 2020-01-23 at 11 27 50](https://user-images.githubusercontent.com/4357962/72980923-71829b00-3dd3-11ea-9902-62ac0442530f.png)